Vagrantfile

```diff
@@ -3,7 +3,7 @@
 
 Vagrant.configure("2") do |config|
   config.vm.provision "ansible_local" do |ansible|
-    ansible.playbook = "vagranttest.yml"
+    ansible.playbook = "molecule/default/tests/vagranttest.yml"
```

Comment:
Is there a reason we are moving the playbook inside molecule's python test directory?

Reply:
Oh, its not really moving
This PR introduces a vagrant test playbook.
I placed this playbook in the molecule test directory, because I wanted to place all tests in a central location.

Where would you propose this be?